setup.py

```diff
@@ -92,5 +89,9 @@ def walk_subpkg(name):
     extras_require={"testing": ["pytest"]},
     install_requires=install_requires,
     zip_safe=False,
-    cmdclass=versioneer.get_cmdclass(),
+    use_scm_version={
```

Comment:
I haven't used scm before, so I can't really comment on this. I've been reading some docs, and they say to add `setup_requires=['setuptools_scm'],` here as well. https://github.com/pypa/setuptools_scm/#setuppy-usage If you're sure that this works as is then let's go ahead.

Reply:
That is correct but my next PR would remove it b/c I'm planning on adding a pyproject.toml to make folium pep 517/518 compliant.

Reply:
Sounds good, so this is merely an intermediate change. Go ahead then.